src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -1344,7 +1344,9 @@ int fx_muxer_t::handle_cli(
     {
         assert(argc > 1);
         if (pal::strcasecmp(_X("-h"), argv[1]) == 0 ||
-            pal::strcasecmp(_X("--help"), argv[1]) == 0)
+            pal::strcasecmp(_X("--help"), argv[1]) == 0 ||
+            pal::strcasecmp(_X("-?"), argv[1]) == 0 ||
+            pal::strcasecmp(_X("/?"), argv[1]) == 0)
```

Comment:
I don't believe we support `/` arguments anywhere else, why start here/now? Especially considering the recent issues we've had with `/`:

https://github.com/dotnet/cli/pull/8748
https://github.com/dotnet/cli/issues/10014
etc.

Reply:
ok, fine.  😄